bindings/c/src/lib.rs

```diff
@@ -472,13 +472,19 @@ pub unsafe extern "C" fn libsql_query(
             return 1;
         }
     };
+    let start_time = std::time::Instant::now();
     let conn = conn.get_ref();
     match RT.block_on(conn.query(sql, ())) {
         Ok(rows) => {
+            let end_time = start_time.elapsed(); // Calculate elapsed time
+            println!("Query execution time: {:?}", end_time);
+            
             let rows = Box::leak(Box::new(libsql_rows { result: rows }));
             *out_rows = libsql_rows_t::from(rows);
         }
         Err(e) => {
+            let end_time = start_time.elapsed(); // Calculate elapsed time
+            println!("Query execution failed. Time elapsed: {:?}", end_time);
```

Comment:
It is a bad practise to print like that in libraries. Please use tracing instead.